docs/src/content/docs/hi/guides/components.mdx

```diff
@@ -296,3 +278,118 @@ import importedCode from '/src/env.d.ts?raw';
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### फ़ाइल वृक्ष
+
+फ़ाइल आइकन और बंधने योग्य उप-निर्देशिकाओं के साथ निर्देशिका की संरचना प्रदर्शित करने के लिए `<FileTree>` घटक का उपयोग करें।
+
+`<FileTree>` के अंदर एक [अव्यवस्थित Markdown सूची](https://www.markdownguide.org/basic-syntax/#unordered-lists) के साथ अपनी फ़ाइलों और निर्देशिकाओं की संरचना निर्दिष्ट करें। नेस्टेड सूची का उपयोग करके एक उप-निर्देशिका बनाएं या इसे विशिष्ट सामग्री के बिना निर्देशिका के रूप में प्रस्तुत करने के लिए सूची आइटम के अंत में `/` जोड़ें।
+
+फ़ाइल वृक्ष के स्वरूप को अनुकूलित करने के लिए निम्नलिखित सिंटैक्स का उपयोग किया जा सकता है:
+
+- किसी फ़ाइल या निर्देशिका का नाम बोल्ड करके उसे हाइलाइट करें, उदा. `**README.md**` पढ़ें.
+- किसी फ़ाइल या निर्देशिका में नाम के बाद और टेक्स्ट जोड़कर एक टिप्पणी जोड़ें।
+- नाम के रूप में `...`. या `…` का उपयोग करके प्लेसहोल्डर फ़ाइलें और निर्देशिकाएँ जोड़ें।
+
+```mdx
+# src/content/docs/example.mdx
+
+import { FileTree } from '@astrojs/starlight/components';
+
+<FileTree>
+
+- astro.config.mjs an **important** file
+- package.json
+- README.md
+- src
+  - components
+    - **Header.astro**
+  - …
+- pages/
+
+</FileTree>
+```
+
+उपरोक्त कोड पृष्ठ पर निम्नलिखित उत्पन्न करता है:
+
+import { FileTree } from '@astrojs/starlight/components';
+
+<FileTree>
+
+- astro.config.mjs एक **महत्वपूर्ण** फ़ाइल
+- package.json
+- README.md
+- src
+  - components
+    - **Header.astro**
+  - …
+- pages/
+
+</FileTree>
+
+### चरण
+
+कार्यों की क्रमांकित सूचियों को स्टाइल करने के लिए `<चरण>` घटक का उपयोग करें।
+यह अधिक जटिल चरण-दर-चरण मार्गदर्शिकाओं के लिए उपयोगी है जहां प्रत्येक चरण को स्पष्ट रूप से हाइलाइट करने की आवश्यकता होती है।
+
+`<चरण>` को मानक Markdown आदेशित सूची के चारों ओर लपेटें।
+सभी सामान्य Markdown सिंटैक्स `<Steps>` के अंदर लागू होते हैं।
+
+````mdx title="src/content/docs/example.mdx"
+import { Steps } from '@astrojs/starlight/components';
+
+<Steps>
+
+1. Import the component into your MDX file:
+
+   ```js
+   import { Steps } from '@astrojs/starlight/components';
+   ```
+
+2. Wrap `<Steps>` around your ordered list items.
+
+</Steps>
+````
+
+उपरोक्त कोड पृष्ठ पर निम्नलिखित उत्पन्न करता है:
+
+import { Steps } from '@astrojs/starlight/components';
+
+<Steps>
+
+1. Import the component into your MDX file:
+
+   ```js
+   import { Steps } from '@astrojs/starlight/components';
+   ```
+
+2. Wrap `<Steps>` around your ordered list items.
+
+</Steps>
```

Comment:
Up to you @at-the-vr, but the language in the example could also be translated to Hindi here.

Reply:
Wow can't believe I omitted that part out 🤦 . Resolved with latest push 👍